ui/src/app/pages/access/two-factor-auth-modal.spec.tsx

```diff
@@ -1,14 +1,19 @@
 import * as React from 'react';
-import { mount } from 'enzyme';
+
+import { render, RenderResult } from '@testing-library/react';
 
 import { TwoFactorAuthModal } from './two-factor-auth-modal';
 
 describe('TwoFactorAuthModal', () => {
-  const component = () =>
-    mount(<TwoFactorAuthModal onCancel={() => {}} onClick={() => {}} />);
+  const component = (): RenderResult =>
+    render(
+      <div id='popup-root'>
```

Comment:
Added this, because our Modal component has a line that is passing:
`    <ReactModal
      parentSelector={() => document.getElementById('popup-root')}`
This parentSelector is called during cleanup.